router.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const router = express.Router();
+const db = require('./data/db');
+
+router.route('/users')
+    .get((req, res) => {
+        db
+            .find()
+            .then(response => res.status(200).json(response))
+            .catch(err => res.status(500).json({ error: "The users information could not be retrieved." }));
+    })
+    .post((req, res) => {
+        const {name, bio} = req.body;
+        if(!name || !bio) res.status(400).json({ errorMessage: "Please provide name and bio for the user." });
+        db
+            .insert({
+                name,
+                bio,
+                created_at: new Date().toString(),
```

Comment:
The database would generate the create_at and updated_at so you wouldn't need to do it yourself